xml/System.Security.Cryptography.X509Certificates/X509Store.xml

```diff
@@ -516,11 +520,12 @@
         <Parameter Name="flags" Type="System.Security.Cryptography.X509Certificates.OpenFlags" Index="2" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="storeName">To be added.</param>
-        <param name="storeLocation">To be added.</param>
-        <param name="flags">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="storeName">The name of the certificate store.</param>
```

Comment:
Should this be "One of the enumeration values that specifies..."?

Reply:
No, this is the string overload.